python/cudf/cudf/utils/utils.py

```diff
@@ -56,10 +56,19 @@ def check_equals_int(a, b):
     return a == b
 
 
-def scalar_broadcast_to(scalar, size, dtype=None):
+def scalar_broadcast_to(scalar, size, dtype=None, nan_as_null=False):
```

Comment:
The only possible issue I am seeing is as cudf Frame that converts any given column with `NaN` to `null` by default, but this will not be the case if you do it with Scalar, which kind of deviates from the default behavior we are following. And I have kept the default as `False` because this is an internal function and needs inherit the need from upper APIs.